llvm/test/CodeGen/AMDGPU/spill-regpressure-less.mir

```diff
@@ -0,0 +1,353 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 3
+# RUN: llc -march=amdgcn -mcpu=gfx90a -start-before=machine-scheduler -stop-after=machine-scheduler -verify-machineinstrs -o - %s | FileCheck -check-prefix=GCN %s
```

Comment:
-run-pass=machine-scheduler should work. also don't need -verify-machineinstrs, used -verify-misched to make sure we get a verify run with LiveIntervals